### DISCUSSION TITLE
PC freeze only when input is present

### DISCUSSION THREAD

**Discussion by ankomen2** [nefarius/ViGEmBus#127] [Q&A]
I am using DS4Windows. After the VIGEmBus driver is live (it's active in device manager) my pc freezes all functions for a second only when an input from the controller is present.

Steps to reproduce the behavior (example):
1. Start DS4Windows
2. Connect PS4 controller
3. Move the stick of the controller or any other input
4. PC freezes for 1 second.

**System details (please complete the following information):**
 - OS: Windows 11 Pro - 22000.160
 - Feeder: DS4Windwos ( also tried with x360ce )
 - Driver Version:  v1.17.333

**Additional context**
I tried everything from restarting the pc to removing everything and starting over again . Different versions of the driver, different versions of DS4Windows.

**Comment by jhonkazuto** [comment]
managed to solve? I have the same problem.
